internal/adapters/operation/operation_flow_redis_test.go

```diff
@@ -270,3 +274,159 @@ func TestWatchOperationCancellationRequests(t *testing.T) {
 		}, 5*time.Second, 100*time.Millisecond)
 	})
 }
+
+func TestListSchedulerPendingOperationIDs(t *testing.T) {
+	type args struct {
+		ctx           context.Context
```

Comment:
The `ctx` as a less important argument that doesn't matter much to our test is commonly created inside the test. What do you think to remove this variable from the table?